recipes/TIMIT/ASR/transducer/train_wav2vec.py

```diff
@@ -0,0 +1,376 @@
+#!/usr/bin/env/python3
+"""Recipe for training a phoneme recognizer with
+Transducer loss on the TIMIT dataset.
+
+To run this recipe, do the following:
+> python train.py hparams/train.yaml --data_folder /path/to/TIMIT
+
+
+Authors
+ * Abdel Heba 2020
+ * Mirco Ravanelli 2020
+ * Ju-Chieh Chou 2020
+"""
+import os
+import sys
+import torch
+import logging
+import speechbrain as sb
+from hyperpyyaml import load_hyperpyyaml
+from speechbrain.utils.distributed import run_on_main
+
+logger = logging.getLogger(__name__)
+
+
+# Define training procedure
+class ASR_Brain(sb.Brain):
+    def compute_forward(self, batch, stage):
+        "Given an input batch it computes the phoneme probabilities."
+        batch = batch.to(self.device)
+        wavs, wav_lens = batch.sig
+        phns, phn_lens = batch.phn_encoded
+
+        # Adding optional augmentation when specified:
+        if stage == sb.Stage.TRAIN:
+            if hasattr(self.hparams, "env_corrupt"):
```

Comment:
I don't see any env_corr on the YAML. Maybe simply remove all references in the train ?

Reply:
He took it out temporarily to save memory, see his comment above